src/app/users/store/users.reducer.ts

```diff
@@ -0,0 +1,23 @@
+import { IUser } from '../user.interface';
+import * as UsersActions from './users.actions';
+export interface State {
+  users: IUser[];
+}
+const initialState: State = {
```

Comment:
change the name to usersInitialState.

also change the name of the interface.

IUsersInitialState

All the interfaces you define, Start with I as OUR naming covention.

Reply:
done°°